webapp/src/components/CommentCard/index.js

```diff
@@ -0,0 +1,240 @@
+import React, { useEffect, useState } from 'react'
+import Box from '@mui/material/Box'
+import Grid from '@mui/material/Grid'
+import IconButton from '@mui/material/IconButton'
+import FilterListIcon from '@mui/icons-material/FilterList'
+import Typography from '@mui/material/Typography'
+import { makeStyles } from '@mui/styles'
+import ThumbUpIcon from '@mui/icons-material/ThumbUp'
+import ThumbDownIcon from '@mui/icons-material/ThumbDown'
+import MenuItem from '@mui/material//MenuItem'
+import Menu from '@mui/material//Menu'
+import Moment from 'react-moment'
+import { useLazyQuery } from '@apollo/client'
+import { useSharedState } from '../../context/state.context'
```

Comment:
move this lines to the second import group

Remember, the first import group is just for third party packages